resources/js/views/RootContainer.vue

```diff
@@ -1,29 +1,38 @@
 <template>
     <div class="root-container">
         <Nav />
-
+        <WelcomeBanner v-bind:showEmailConfirmedBanner="showEmailConfirmedBanner" />
```

Comment:
you can remove "v-bind" as `:` will do the same

you could also pick a smaller prop name, this was an example for illustration